plugins/polio/api/vaccines/stock_management.py

```diff
@@ -876,7 +939,12 @@ class VaccineStockManagementViewSet(ModelViewSet):
 
     """
 
-    permission_classes = [VaccineStockManagementReadWritePerm]
+    permission_classes = [
+        lambda: VaccineStockManagementPermission(
+            non_admin_perm=permission.POLIO_VACCINE_STOCK_MANAGEMENT_READ,
+            admin_perm=permission.POLIO_VACCINE_STOCK_MANAGEMENT_WRITE,
```

Comment:
It's confusing that you put them in "non-admin, admin" order where in the constructor you did "admin, non-admin."

This should be consistent